Formula/resty.rb

```diff
@@ -1,8 +1,8 @@
 class Resty < Formula
   desc "Command-line REST client that can be used in pipelines"
   homepage "https://github.com/micha/resty"
-  url "https://github.com/micha/resty/archive/2.2.tar.gz"
-  sha256 "a6961e16f1489ffa25bf9f2d8b38b9599c008d49267e32adf93f4f87184d4857"
+  url "https://github.com/micha/resty/archive/2.3.zip"
```

Comment:
Use the `.tar.gz` archive, not the `.zip` one, as source. This may help with permission issues.

Reply:
okey dokey

Reply:
@fxcoudert done :)